Intro_markdown.Rmd

```diff
@@ -225,30 +225,40 @@ offenders <- readr::read_csv("Offenders_Chicago_Police_Dept_Main.csv")
 
 Note that the above assumes that the csv file is in your working directory, otherwise you will need to include the file path - see section 2.1.
 
+Note that this code is only suitable for csv files, so it is assumed by default that the first line of the file contains a header (header = T) and the columns are separated by a comma symbol (sep = ",").
+
+There are other commands and various packages that can be used to import datasets with other extensions (e.g. .xls) e.g. see http://www.statmethods.net/input/importingdata.html.
+
+
 ### Importing data from the Analytical Platform cloud storage (Amazon S3)
 
-#### Importing data using the `botor` package {-}
+Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. There are two options for packages that can be used to import data from S3: `Rs3tools` and `botor`.
 
-Data that has been approved for storage on the Analytical Platform is generally stored in a data source (referred to as a 'bucket') on Amazon S3, which is the cloud storage solution used by the Analytical Platform. To import the data for this session from the 'alpha-r-training' S3 bucket use this command:
+#### Importing data using the `Rs3tools` package {-}
+
+The `Rs3tools` package is maintained by other analysts in MoJ and has the advantage of being R-native, meaning it is simpler to install and use compared to `botor`. Here's how it can be installed:
```

Comment:
How about amending to:
The `Rs3tools` package is maintained by other analysts in MoJ and has the advantage of being R-native, resulting in it being quicker to install than `botor` which requires a python environment. Here's how it can be installed: